initial-promises/node-server/common/1.0/update.cf

```diff
@@ -318,11 +318,14 @@ body action u_background
 ############################################
 body classes success(if, else, kept)
 {
-        promise_kept => { "${kept}" };
-        promise_repaired => {"${if}"};
-        repair_failed => { "${else}" };
-        repair_denied => { "${else}" };
-        repair_timeout => { "${else}" };
+        promise_kept     => { "${kept}" };
+        promise_repaired => { "${if}"   };
+        repair_failed    => { "${else}" };
+        repair_denied    => { "${else}" };
+        repair_timeout   => { "${else}" };
+
+        cancel_kept      => { "${else}" };
+        cancel_repaired  => { "${else}" };
```

Comment:
no, the code is clever, and report the worse state (repaired in case of success and repaired, error in case of repaired and error, etc)